app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditViewModel.kt

```diff
@@ -384,24 +404,46 @@ class VaultAddEditViewModel @Inject constructor(
     private fun registerFido2Credential(
         request: Fido2CredentialRequest,
         content: VaultAddEditState.ViewState.Content,
+    ) {
+        val createOptions =
+            fido2CredentialManager.getPasskeyCreateOptionsOrNull(request.requestJson)
```

Comment:
This can be formatted cleaner:

```kotlin
val createOptions = fido2CredentialManager
    .getPasskeyCreateOptionsOrNull(request.requestJson)
    ?: run {
```